01-algorithms/09-algorithms-class-discussion.md

```diff
@@ -0,0 +1,22 @@
+# Class Discussions
+
+## C18 Recordings
+The following videos are from the class discussions from C18. There are two classrooms, Tigons and Ligers. While the material taught to the two classes was the same, there may be different questions asked by students in the classes.
```

Comment:
This is not necessary, but it might read a little smoother if the first couple sentences are combined, something like:

"The following videos are the live session recordings of C18’s Tigons and Ligers classrooms. Though the same material was taught to the two classes, there may be different questions asked by students in each class."

Reply:
I like that much better! I'll make that update.